homeworks/olha.tymchyshyn_olyatmchshn/7-arrays-loops/arrays.js

```diff
@@ -0,0 +1,80 @@
+const arrayOne = [3, 0, -5, 1, 44, -12, 3, 0, 0, 1, 2, -3, -3, 2, 1, 4, -2, -3, -1];
+const arrayTwo = [-1, -8, -2];
+const arrayThree = [1, 7, 3];
+const arrayFour = [1, undefined, 3, 5, -3];
+const arrayFive = [1, NaN, 3, 5, -3];
+
+function maxNumberOfArr(array) {
+    let maxNum = array[0];
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] > maxNum) {
+            maxNum = array[i];
+        }
+    }
+    return maxNum;
+    // write your code here and instead of array return maxNumber
+}
+
+console.log(maxNumberOfArr(arrayOne)); // 44
+console.log(maxNumberOfArr(arrayTwo)); // -1
+console.log(maxNumberOfArr(arrayThree)); // 7
+console.log(maxNumberOfArr(arrayFour)); // 5
+console.log(maxNumberOfArr(arrayFive)); // 5
+
+function minNumberOfArr(array) {
+    let minNum = array[0];
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] < minNum) {
+            minNum = array[i];
+        }
+    }
+    return minNum;
+    // write your code here and instead of array return minNumber
+}
+
+console.log(minNumberOfArr(arrayOne)); // -12
+console.log(minNumberOfArr(arrayTwo)); // -8
+console.log(minNumberOfArr(arrayThree)); // 1
+console.log(minNumberOfArr(arrayFour)); // -3
+console.log(minNumberOfArr(arrayFive)); // -3
+
+function sumNumbersOfArr(array) {
+    let sum = 0;
+    for (let i = 0; i < array.length; i++) {
+        if (typeof array[i] === 'number' && !Number.isNaN(array[i])) {
+            sum += array[i];
+        }
+    }
+    return sum;
+    // write your code here and instead of array return sum of numbers
+}
+
+console.log(sumNumbersOfArr(arrayOne)); // 32
+console.log(sumNumbersOfArr(arrayTwo)); // -11
+console.log(sumNumbersOfArr(arrayThree)); // 11
+console.log(sumNumbersOfArr(arrayFour)); // 6
+console.log(sumNumbersOfArr(arrayFive)); // 6
+
+function onlyNegativeNumbers(array) {
+    const negativeNumbers = array.filter((number) => number < 0);
+    return negativeNumbers;
+    // write your code here and return array only with negative numbers
+}
+
+console.log(onlyNegativeNumbers(arrayOne)); // [-5, -12, -3, -3, -2, -3, -1]
+console.log(onlyNegativeNumbers(arrayTwo)); // [-1, -8, -2]
+console.log(onlyNegativeNumbers(arrayThree)); // []
+console.log(onlyNegativeNumbers(arrayFour)); //  [-3]
+console.log(onlyNegativeNumbers(arrayFive)); // [-3]
+
+function onlyPositiveNumbers(array) {
+    const positiveNumbers = array.filter((number) => number > 0);
```

Comment:
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/d043c66d-8df4-4e8a-8c7d-d6bddc92c9f5)
Common approach; you set something to variable, if you like to use it twice at least.
You can use `=>` single line function